config/resolve_credentials_test.go

```diff
@@ -278,7 +370,15 @@ func TestSharedConfigCredentialSource(t *testing.T) {
 			}
 
 			if e, a := c.expectedSecretKey, creds.SecretAccessKey; e != a {
-				t.Errorf("expected %v, but received %v", e, a)
+				t.Errorf("expect %v, but received %v", e, a)
+			}
+
+			if e, a := c.expectedSessionToken, creds.SessionToken; e != a {
+				t.Errorf("expect %v, got %v", e, a)
+			}
+
+			if cleanup != nil {
+				cleanup()
```

Comment:
Should we use a `defer cleanup()` instead? we have few `t.Fatalf` calls above.

Reply:
good catch